<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayListWidget</class>
 <widget class="QWidget" name="PlayListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste de lecture</string>
  </property>
  <property name="windowOpacity">
   <double>0.920000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbMy">
     <property name="title">
      <string>Mes videos</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="myList"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddFolder">
        <property name="text">
         <string>Ajouter un dossier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCurrent">
     <property name="title">
      <string>PlayList courante</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="currentList"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPlayList">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unsaved playlist</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Sauvegarder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Charger</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
